ReactAndroid/src/main/jni/Application.mk

```diff
@@ -4,7 +4,13 @@ APP_ABI := armeabi-v7a x86
 APP_PLATFORM := android-9
 
 APP_MK_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
-NDK_MODULE_PATH := $(APP_MK_DIR):$(THIRD_PARTY_NDK_DIR):$(APP_MK_DIR)/first-party
+
+# Use ; instead of : as path delimiter on Windows
+ifeq ($(OS),Windows_NT)
+NDK_MODULE_PATH := $(APP_MK_DIR);$(THIRD_PARTY_NDK_DIR);$(APP_MK_DIR)first-party
+else
+NDK_MODULE_PATH := $(APP_MK_DIR):$(THIRD_PARTY_NDK_DIR):$(APP_MK_DIR)first-party
+endif
```

Comment:
We should use $(HOST_DIRSEP) instead of doing it this way. Do you mind updating this?